src/sensor_trigger.cpp

```diff
@@ -48,8 +49,39 @@ SensorTrigger::SensorTrigger(const rclcpp::NodeOptions & node_options)
     return;
   }
 
+  if (cpu_ < 0 || cpu_ >= static_cast<int>(std::thread::hardware_concurrency())) {
+    RCLCPP_WARN_STREAM(
```

Comment:
```suggestion
    RCLCPP_ERROR_STREAM(
```
or 
```suggestion
    RCLCPP_FATAL_STREAM(
```
If this is not the intended one, I'd appreciate your consideration of changing the log level because the node seems to exit if the condition here is fulfilled.

Reply:
The reason I'm using RCLCPP_WARN_STREAM here, is because the node will continue to work, just without schedule priority / cpu affinity. I think RCLCPP_ERROR_STREAM would also be appropriate, but also seems harsh considering there is no option to turn off schedule priority and cpu affinity.

Reply:
If the node is intended to continue working, the bunch of codes
```c++
    rclcpp::shutdown();
    return;
```
([l. 57-58](https://github.com/tier4/sensor_trigger/blob/ef9879a1639298e7fb9f2fc5700d078d2bf5a15e/src/sensor_trigger.cpp#L57-L58)) just after `RCLCPP_WARN_STREAM` is correct? It looks to me that the constructor will finish without creating a thread to execute `run` function if the node meets the condition to show here dialogue.

Reply:
Thanks, I tidied this up with [0e982da](https://github.com/tier4/sensor_trigger/pull/22/commits/0e982da929c67b92c244081dd2c3aeebf5238b28) and [20ee150](https://github.com/tier4/sensor_trigger/pull/22/commits/20ee1504865a4605ddf9316a4a60bc9c1978d6a1).

Reply:
Thank you for the prompt resolving. LGTM!